src/unit_tests/wrappers/wazuh/syscheckd/create_db_wrappers.c

```diff
@@ -64,3 +64,5 @@ int __wrap_fim_whodata_event(whodata_evt * w_evt)
 #endif
     return 1;
 }
+
+void __wrap_free_entry(fim_entry *entry) {}
```

Comment:
Is this wrapper necessary? If it is, please add an empty return to indicate that the function does nothing.
```C
void __wrap_free_entry(fim_entry *entry) {
    return;
}
```

The `entry` parameter is unused:
```
unit_tests/wrappers/wazuh/syscheckd/create_db_wrappers.c: In function ‘__wrap_free_entry’:
unit_tests/wrappers/wazuh/syscheckd/create_db_wrappers.c:68:35: warning: unused parameter ‘entry’ [-Wunused-parameter]
   68 | void __wrap_free_entry(fim_entry *entry) {}
      |                        ~~~~~~~~~~~^~~~~
```